### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by CloneWith** [CloneWith/opsu#101] [Announcements]
## What's Changed

* Code refactoring
* Custom interface color support (may need a full restart)
* Use *Real* Auto mod (**staging**, fails on no health left)
* Remove restrictions for beatmap difficulty settings (feel free to try some insane settings!)
* Add option to always use the builtin Back button style (it does seems good!)
* Minor changes and improvements

**Full Changelog**: https://github.com/CloneWith/opsu/compare/v1.1.0...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/CloneWith/opsu/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by Zinoujoker** [comment]
Good keep up 🆙 👍 😅
